sdk/trace/src/main/java/io/opentelemetry/sdk/trace/RecordEventsReadableSpan.java

```diff
@@ -519,7 +518,7 @@ int getTotalRecordedLinks() {
   }
 
   @GuardedBy("lock")
-  private ReadableAttributes getImmutableAttributes() {
+  private Attributes getImmutableAttributes() {
```

Comment:
now that the ordering requirement on attributes has been relaxed, we could go back to returning our array-backed ones here, if we wanted to. Not a change that needs to be in this PR.

Reply:
Yup noticed this too :)